src/main/java/com/cegeka/academy/domain/Category.java

```diff
@@ -18,7 +18,8 @@
     @Column(name = "description")
     private String description;
 
-    @ManyToMany(mappedBy = "categories")
+    @ManyToMany(fetch = FetchType.LAZY, cascade = {CascadeType.ALL},
```

Comment:
Nu stiu daca e bine sa ai  aici `CascadeType.ALL`. De regula se foloseste in entitatea *owner / parinte*.